raven/contrib/tornado/__init__.py

```diff
@@ -11,6 +11,7 @@
 
 from tornado import ioloop
 from tornado.httpclient import AsyncHTTPClient, HTTPError
+from tornado import web
```

Comment:
I did it this way because of ``tornado.httpclient.HTTPError`` (see line above), which isn't the same as ``tornado.web.HTTPError`` (the naming is rather confusing...). Therefore using ``web.HTTPError`` makes it clear which one is used. Maybe ``from tornado.web import HTTPError as HTTPServerError`` and change the line above to ``HTTPError as HTTPClientError`` would be better?